MovieCave - MVVM/Movie-Cave/Movie-Cave/View/MovieDetailsView/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,204 @@
+//
+//  MovieDetailsViewController.swift
+//  Movie-Cave
+//
+//  Created by Admin on 3.09.23.
+//
+
+import UIKit
+import Combine
+import AVKit
+import AVFoundation
+
+class MovieDetailsViewController: UIViewController {
+    
+    //MARK: - IBOutlets
+    @IBOutlet private weak var trailerStackView: UIStackView!
+    @IBOutlet private weak var bigPosterImageView: UIImageView!
+    @IBOutlet private weak var castStackView: UIStackView!
+    @IBOutlet private weak var genreView: UIView!
+    @IBOutlet private weak var favoritesStarLabel: UILabel!
+    @IBOutlet private weak var movieYearLable: UILabel!
+    @IBOutlet private weak var movieTitleLabel: UILabel!
+    @IBOutlet private weak var describtionImage: UIImageView!
+    @IBOutlet private weak var describtionLabel: UILabel!
+    
+    //MARK: - Properties
+    private var trailerURLs: [String] = ["url1", "url2", "url3", "url3", "url3"]
+    private var castArray: [Cast] = []
+    private var genres: [Genres] = []
+    private var viewModel: MovieDetailsViewModelProtocol?
```

Comment:
having three viewControllers with 3 different by functionality viewmodels but with same name is confusing, rename